### PR TITLE
Update OrgAuth StarterWeb for runtime changes

### PR DESCRIPTION
**Pull request by phenning**
@Tratcher @rustd 

Diff is a bit odd due to reorganization of the directory structure for templates to accommodate the AD read version of Startup.cs  (The NoRead one is differed from the original version, the Read one is being displayed as a new file)

### CODE REVIEW

File: src/Rules/StarterWeb/OrganizationalAuth/Multiple/Common/Controllers/AccountController.cs [hunk 1]
```diff
@@ -17,13 +17,11 @@ public IActionResult SignIn()
                 new AuthenticationProperties { RedirectUri = "/" }, OpenIdConnectDefaults.AuthenticationScheme);
         }
 
-        public async Task<IActionResult> SignOut()
+        public IActionResult SignOut()
```
Comment (rustd) [inline src/Rules/StarterWeb/OrganizationalAuth/Multiple/Common/Controllers/AccountController.cs:20]: Can you please make these changes for MultIOrg templates as well?

File: src/Rules/StarterWeb/OrganizationalAuth/Single/Common/Controllers/AccountController.cs [hunk 2]
```diff
@@ -17,13 +17,11 @@ public IActionResult SignIn()
                 new AuthenticationProperties { RedirectUri = "/" }, OpenIdConnectDefaults.AuthenticationScheme);
         }
 
-        public async Task<IActionResult> SignOut()
```
Comment (rustd) [inline src/Rules/StarterWeb/OrganizationalAuth/Single/Common/Controllers/AccountController.cs:20]: Can you please make this change for MultiOrg template as well?

File: src/Rules/WebAPI/OrganizationalAuth/Single/Startup.cs [hunk 3]
```diff
@@ -52,7 +52,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
             app.UseJwtBearerAuthentication(new JwtBearerOptions
             {
                 Authority = Configuration["Authentication:AzureAd:AADInstance"] + Configuration["Authentication:AzureAd:TenantId"],
-                Audience = Configuration["Authentication:AzureAd:AudienceUrl"]
```
Comment (rustd) [inline src/Rules/WebAPI/OrganizationalAuth/Single/Startup.cs:55]: I am assuming there will be a Org Tooling/ publish update for these settings.

Comment (rustd) [inline src/Rules/WebAPI/OrganizationalAuth/Single/Startup.cs:55]: For Web API, this covers both Read and NonRead cases?

File: src/Rules/StarterWeb/OrganizationalAuth/Multiple/Common/Controllers/AccountController.cs (hunk 1 above)
Comment (phenning) [inline src/Rules/StarterWeb/OrganizationalAuth/Multiple/Common/Controllers/AccountController.cs:20]: Already done, this is the MultiOrg. :)

File: src/Rules/StarterWeb/OrganizationalAuth/Single/Common/Controllers/AccountController.cs (hunk 2 above)
Comment (phenning) [inline src/Rules/StarterWeb/OrganizationalAuth/Single/Common/Controllers/AccountController.cs:20]: Done, see above

File: src/Rules/WebAPI/OrganizationalAuth/Single/Startup.cs (hunk 3 above)
Comment (phenning) [inline src/Rules/WebAPI/OrganizationalAuth/Single/Startup.cs:55]: Correct, there will a corresponding Tooling update here as well.

Comment (phenning) [inline src/Rules/WebAPI/OrganizationalAuth/Single/Startup.cs:55]: Yes, this should cover both read and non read, as currently for web api they are identical

**Comment by rustd** [comment]
:shipit:

**Comment by phenning** [comment]
Merged

**Pull request closed without merging**